ruby/basics/variables_strings_and_numbers.md

```diff
@@ -1,119 +1,371 @@
 
 # Variables, Strings and Numbers
-A brief summary about what this lesson is about and why the topics or concepts it covers are important.
+Since you've already done a decent chunk of Ruby in the prep course, this should be a healthy refresher of what you've already learned with Variables, Strings and Numbers.
+
+This lesson will build on your knowledge and go much deeper and further than you went before.
+
+Just like you can't build a castle on a weak foundation, neither can you write a great program without having strong knowledge of the more basic data types. If you want to write the next [Facebook](www.facebook.com) or [LingsCars](www.lingscars.com) you'll need a fundamental knowledge of basic data types and what you can do with them.
+
+Variables, Strings and Numbers can be a little unappreciated next to much cooler Ruby features like Blocks and Hashes but they are so fundamental to Programming that you should know them inside out and what you can do with them before you can really master more advanced features.
```

Comment:
Nit. I find some unnecessary capitalization in this sentence. Do we need to capitalize 'Programming'?